samples/bluetooth/channel_sounding_ras_initiator/prj.conf

```diff
@@ -12,6 +12,7 @@ CONFIG_BT_SMP=y
 CONFIG_BT_CENTRAL=y
 CONFIG_BT_MAX_CONN=1
 CONFIG_BT_BONDABLE=n
+CONFIG_HEAP_MEM_POOL_SIZE=1024
```

Comment:
This shouldn't be required as we use this to add heap here https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/bluetooth/services/ras/rreq/Kconfig.ras_rreq#L27